REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.5
-Compat 0.8
+Compat 0.18
```

Comment:
doesn't look like any of the type aliases are parametric, so 0.17 should be enough